### ISSUE TITLE
Stereo in jitsimeet

### ISSUE THREAD

**Issue by grishakonzalaev** [jitsi/jitsi-meet#3180]
It was very cool to see stereo in jitsimeet earlyer. Now it's mono sound. Why?

**Comment by damencho** [comment]
Are you talking about meet.jit.si or your own deployment?
If it is your own deployment do you have stereo: true in your config?

Please in the future do not open new issues before asking questions on our community forum: https://community.jitsi.org.

**Comment by grishakonzalaev** [comment]
This config is not working.
I test it at original not deploiment jitsi meet.

**Comment by damencho** [comment]
So you mean you were testing on meet.jit.si. I see the config there is false and I will ask around why is that and if that had changed at some point.

**Comment by grishakonzalaev** [comment]
I want stereo for streaming some sounds for my friends from my computer. It's easy make through stereo mixer.
Also stereo need in android phones. Almost phones have 2 or more microphones.

**Comment by a11cf0** [comment]
Hello.
I'd also like to see stereo support in Jitsi Meet as it is useful on devices with two or more microphones, e.g. most modern smartphones, and for occasional sound streaming.
I think it was possible to get stereo working sometime ago, but it seems to no longer be so.
I tested all audio-related options I could find both on  meet.jit.si by passing URL parameters and on my own deployment by editing config.js.
Here's what I have tried to change in my config.js so far without any effect:
```
disableAEC: true,
disableAGC: true,
disableAP: true,
disableHPF: true,
disableNS: true,
stereo: true, // Manually added this one because it's not present in the config.
startBitrate: 300000 // Just for fun.
```
I'm using the latest stable Jitsi Meet Debian packages from the official Jitsi repo.
